tests/src/test/scala/ch/renku/acceptancetests/MinimalFunctionalitySpec.scala

```diff
@@ -47,4 +54,15 @@ class MinimalFunctionalitySpec
     launchAnonymousEnvironment map (_ => stopEnvironment(anonEnvConfig.projectId))
 
   }
+
+  def addChangeToProject(implicit projectDetails: ProjectDetails, docsScreenshots: DocsScreenshots): Unit = {
+    val projectPage    = ProjectPage()
+    val jupyterLabPage = launchEnvironment
```

Comment:
Since launchEnvironment requires a DocsScreenshots object, I suggest changing this line to the following:
```
    implicit val docsScreenshots = new DocsScreenshots(this, browser) {
      override lazy val captureScreenshots: Boolean = false
    }
    val jupyterLabPage = launchEnvironment
```